src/helper.cc

```diff
@@ -891,38 +890,30 @@ Helper::Client::handleFewerServers(const bool madeProgress)
 }
 
 void
-Helper::Session::HelperServerClosed(Session * const srv)
+Helper::Client::sessionClosed(SessionBase &srv)
```

Comment:
Since we are changing pointer to a reference, let's use modern terminology:

```suggestion
Helper::Client::sessionClosed(SessionBase &session)
```